frontend/packages/client/src/api/leaderboard.js

```diff
@@ -0,0 +1,17 @@
+import { checkResponse } from '../utils';
+import { COMMUNITIES_URL } from './constants';
+
+const LEADERBOARD_SIZE = 10;
+const getLeaderBoardUrl = (communityId) =>
+  `${COMMUNITIES_URL}/${communityId}/leaderboard?count=${LEADERBOARD_SIZE}`;
+
+const fetchLeaderBoard = async (communityId) => {
+  try {
+    const response = await fetch(getLeaderBoardUrl(communityId));
+    return await checkResponse(response);
+  } catch (err) {
+    throw err;
+  }
+};
```

Comment:
I like this approach of adding this in a separate file!

What do you think about making a variable `pageSize` a param to the Url then if that changes on the hook it's already prepared

```suggestion
const LEADERBOARD_SIZE = 10;
const getLeaderBoardUrl = (communityId, pageSize) =>
  `${COMMUNITIES_URL}/${communityId}/leaderboard?count=${pageSize}`;

const fetchLeaderBoard = async (communityId, pageSize = LEADERBOARD_SIZE ) => {
  try {
    const response = await fetch(getLeaderBoardUrl(communityId, pageSize));
    return await checkResponse(response);
  } catch (err) {
    throw err;
  }
};
```

Reply:
Oh yes - that is a better approach.